src/main/java/org/broadinstitute/hellbender/utils/read/FlowBasedReadUtils.java

```diff
@@ -26,17 +28,26 @@ public class FlowBasedReadUtils {
     static public class ReadGroupInfo {
         final public String  flowOrder;
         final public int     maxClass;
-
+        final public boolean isFlowPlatform;
         private String  reversedFlowOrder = null;
 
         public ReadGroupInfo(final SAMReadGroupRecord readGroup) {
-
-            Utils.nonNull(readGroup);
-            this.flowOrder = readGroup.getFlowOrder();
-            Utils.nonNull(this.flowOrder);
-
-            String mc = readGroup.getAttribute(FlowBasedRead.MAX_CLASS_READ_GROUP_TAG);
-            this.maxClass = (mc == null) ? FlowBasedRead.MAX_CLASS : Integer.parseInt(mc);
+            if (readGroup.getPlatform()==null){
+                isFlowPlatform = false;
+            } else {
+                isFlowPlatform = NGSPlatform.valueOf(readGroup.getPlatform()).getSequencerType() == SequencerFlowClass.FLOW;
```

Comment:
Old Ultima data used PL:LS454 because ULTIMA was not a legal platform. I think I prefer allowing LS454 and ULTIMA to work this way given low chance anyone is now running LS454 data

Reply:
Can you special case the PL:LS454 platform and give an explanation about it here? Again, I want to be as specific as possible about the inputs here since this is a greedy non opt-in behavior we should be careful we don't start clobbering somebodies inputs in all cases. If it IS labeled as 454 data can you check for the other fields that should make it readable as ultima data?

Reply:
Added tests